man/docker-daemon.8.md

```diff
@@ -464,6 +465,16 @@ this topic, see
 Otherwise, set this flag for migrating existing Docker daemons to a
 daemon with a supported environment.
 
+Here is supported options of *zfs*:
+
+### zfs.fsname
+
+Set zfs filesystem under which docker will create its own datasets.
+By default docker will pick up the zfs filesystem where docker graph
+(`/var/lib/docker`) is located.
+
+Example use: `docker daemon -s zfs --storage-opt zfs.fsname=zroot/docker`
```

Comment:
should `zroot/docker` be an absolute path? (`/zroot/docker`) - I don't know, haven't used ZFS myself